github.com.hashicorp.go.kms.wrapping.v2.types.proto

```diff
@@ -83,6 +83,10 @@ message KeyInfo {
 
 	// Mechanism specific flags
 	uint64 flags = 6;
+
+	KeyType key_type = 7;
```

Comment:
Can you comment these as they are in the options?

Reply:
done.